docs/content/bundle/manifest/_index.md

```diff
@@ -400,21 +400,29 @@ are automatically defaulted, such as `dry-run`, `help`, `log`, and `status`. You
 actions unless you want to change the defaults.
 
 You may want to declare your custom action when the action does not make any changes, and its execution should not 
-be recorded (`stateless: true` and `modifies: false`). The `help` action is supported out-of-the-box by Porter
-and is automatically defaulted to this definition so you do not need to declare it. If you have an action that is 
-similar to `help`, but has a different name, you should declare it in the `customActions` section.
-
+be recorded (`stateless: true` and `modifies: false`). An example of usecases is:
 ```
 customActions:
-  myhelp:
-    description: "Print a special help message"
+  getLicense:
+    description: "Return the product license information"
     stateless: true
     modifies: false
+
+getLicense:
```

Comment:
Are you trying to demonstrate that this action will be recorded? It's a bit unclear for this example why you are including an output but then not discussing how actions that generate outputs are recorded.

Maybe split this into two custom actions? Keep myhelp as it was, and then add the getLicense one too, along with a bundle output defined so that you can explain why one is recorded and the other one isn't?